src/test/stateful_modelinstance_test.cpp

```diff
@@ -225,57 +231,35 @@ class MockedStatefulModelInstance : public ovms::StatefulModelInstance {
         sequenceManagerLock.unlock();
 
         timer.start("get infer request");
-        ovms::ExecutingStreamIdGuard executingStreamIdGuard(getInferRequestsQueue());
         ovms::ExecutingStreamIdGuard_2 executingStreamIdGuard_2(getInferRequestsQueue_2());
-        InferenceEngine::InferRequest& inferRequest = executingStreamIdGuard.getInferRequest();
         ov::runtime::InferRequest& inferRequest_2 = executingStreamIdGuard_2.getInferRequest();
         timer.stop("get infer request");
 
-        timer.start("preprocess");
```

Comment:
You remove timer start, but leave timer stop for preprocessing. Is that intentional?